styles.css

```diff
@@ -1 +1,70 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+/*mobile grid*/
+.container {
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns:1fr;
+    grid-auto-rows:50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+        "a"
+        "b"
+        "c"
+        "d"
+        "e"
+        "."
+        "f";
+}
+
+/*desktop grid*/
+@media screen and (min-width:400px) {
+    .container {
+        display: grid;
+        grid-gap: 16px;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-auto-rows: 50px 174px 100px auto 50px;
```

Comment:
in de uitwerking van de opdracht staan net wat andere formaten namelijk;
grid-template-rows: 50px 190px 100px auto 50px; Box c in het voorbeeld, bij jou box 3 is 190 want 240 - 50.